azurerm/internal/services/netapp/netapp_volume_data_source.go

```diff
@@ -75,6 +75,32 @@ func dataSourceNetAppVolume() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			"data_protection_replication": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint_type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"remote_volume_location": azure.SchemaLocationForDataSource(),
+
+						"remote_volume_resource_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"replication_schedule": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+					},
+				},
+			},
```

Comment:
as these are in a data sould this should beb computed?

Reply:
Yes, just fixed it.